packages/11ty/content/_data/eleventyComputed.js

```diff
@@ -25,20 +25,39 @@ module.exports = {
     })
   },
   eleventyNavigation: {
+    /**
+     * Explicitly define page data properties used in the TOC
+     * since eleventyNavigation does not include the entire page object
+     */
+    data: (data) => {
+      return {
+        abstract: data.abstract,
+        contributor: data.contributor,
+        figure: data.figure,
+        image: data.image,
+        label: data.label,
+        layout: data.layout,
+        object: data.object,
+        online: data.online,
+        short_title: data.short_title,
+        subtitle: data.subtitle,
+        summary: data.summary,
+        title: data.title,
+        weight: data.weight
+      }
+    },
```

Comment:
The alternative to this is providing the `tableOfContents/list` with `pages` and the navigation object to render and then looking up the full `page` object as we render each TOC item. `eleventyNavigation` items only include select page data (title and url), not everything we need to render quire TOCs which can include contributors, abstracts, etc.

Reply:
@hbalenda let's use this implementation, it is nicely explicit.